spring-integration-core/src/main/java/org/springframework/integration/channel/AbstractMessageChannel.java

```diff
@@ -92,6 +92,10 @@
 
 	private MeterRegistry meterRegistry;
 
+	private Timer successTimer;
```

Comment:
I apply the same optimization to the `NullChannel` as well.

Reply:
thanks